components/logstash/config/logstash.conf

```diff
@@ -55,8 +55,9 @@ output {
             driver_class => 'org.postgresql.Driver'
             connection_string => 'jdbc:postgresql://{{ ctx.instance.runtime_properties.postgresql_host }}:5432/{{ ctx.instance.runtime_properties.postgresql_db_name }}?user={{ ctx.instance.runtime_properties.postgresql_username }}&password={{ ctx.instance.runtime_properties.postgresql_password }}'
             statement => [
-              "INSERT INTO logs (timestamp, execution_fk, logger, level, message, message_code) VALUES(CAST (? AS TIMESTAMP), (SELECT storage_id FROM executions WHERE id = ?), ?, ?, ?, ?)",
+              "INSERT INTO logs (timestamp, deployment_fk, execution_fk, logger, level, message, message_code) VALUES(CAST (? AS TIMESTAMP), (SELECT storage_id FROM deployments WHERE id = ?), (SELECT storage_id FROM executions WHERE id = ?), ?, ?, ?, ?)",
```

Comment:
In the corresponding PR in cloudify-manager I asked why deployment_fk is needed is you this information is available through executions_fk..?